src/configurations/destinations/fb_custom_audience/db-config.json

```diff
@@ -11,13 +11,12 @@
     "saveDestinationResponse": true,
     "includeKeys": ["oneTrustCookieCategories", "consentManagement"],
     "excludeKeys": [],
-    "supportedSourceTypes": ["cloud", "warehouse", "shopify"],
+    "supportedSourceTypes": ["cloud", "warehouse"],
```

Comment:
shopify we want to keep as some customers trying it out

Reply:
Done